src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractPackage`2.cs

```diff
@@ -76,6 +75,12 @@ protected override void Initialize()
                 this.Workspace.StartSolutionCrawler();
             }
 
+            // start remote host if enabled
+            if (this.Workspace.Options.GetOption(InternalFeatureOnOffOptions.RemoteHost))
+            {
+                Workspace.Services.GetService<IRemoteHostService>()?.Enable();
```

Comment:
Is this expensive to do.  Is this a blocking call?  Will this have an impact on packageload?  Can we star tthe remote host asynchronously?